state-chain/custom-rpc/src/lib.rs

```diff
@@ -357,7 +359,13 @@ mod boost_pool_details {
 					.into_iter()
 					.map(|(deposit_id, owed_amounts)| PendingBoost {
 						deposit_id,
-						owed_amounts: map_to_vec(owed_amounts),
+						owed_amounts: owed_amounts
```

Comment:
can inline the other map_to_vec usage now

Reply:
FWIW I did consider inlining it, but decided to keep because `map_to_vec` has a very limited scope and it seemed to have improved readability by de-cluttering the caller IMO. It is subjective, but I will inline so we can make progress.